subsys/bluetooth/host/iso.c

```diff
@@ -28,6 +28,9 @@ NET_BUF_POOL_FIXED_DEFINE(iso_tx_pool, CONFIG_BT_ISO_TX_BUF_COUNT,
 NET_BUF_POOL_FIXED_DEFINE(iso_rx_pool, CONFIG_BT_ISO_RX_BUF_COUNT,
 			  CONFIG_BT_ISO_RX_MTU, NULL);
 
+static struct iso_meta_data iso_data[CONFIG_BT_ISO_RX_BUF_COUNT];
+#define iso_meta(buf) (&iso_data[net_buf_id(buf)])
+
```

Comment:
Not sure I get this, why is this better than just increate the user_data? If we can't increase just for the ISO pool than perhaps that is the real problem here.

Reply:
I agree, but it was suggested that increasing user_data was a bad idea, hence I changed to this approach as requested.